roles/kubernetes-apps/ansible/templates/netchecker-server-deployment.yml.j2

```diff
@@ -35,8 +35,8 @@ spec:
             allowPrivilegeEscalation: false
             capabilities:
               drop: ['ALL']
-            runAsUser: {{ netchecker_server_user | default('0') }}
-            runAsGroup: {{ netchecker_server_group | default('0') }}
+            runAsUser: 1000
```

Comment:
Does keep `{{ netchecker_server_user | default('0') }}` is better.

Reply:
I would scrape the `default('0')` in all cases, because it breaks in CI (that's why I changed that initially)
But yeah, we could keep the variable if it's needed.

Reply:
Hum actually it's unrelated to the cleanup, so I'll scrape that, we'll see that another time.